src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -102,6 +102,14 @@ struct MHAKernel {
     MHAKernel() = delete;
     explicit MHAKernel(GraphContext::CPtr ctx) : context(std::move(ctx)) {}
 
+    static constexpr impl_desc_type getImplType() {
+        if constexpr (KType == ScaledDotProductAttention::KT_REF) {
```

Comment:
I believe we do not need if else in this case.
Just:
static_assert();
return;

Reply:
@EgorDuplensky agree, fixed